shared_model/bindings/bindings.i

```diff
@@ -51,6 +54,19 @@ namespace std {
 %rename(kp_equal) shared_model::crypto::Keypair::operator==;
 %rename(hash_invoke) shared_model::crypto::Hash::Hasher::operator();
 
+%ignore shared_model::interface::PermissionSet::PermissionSet(std::initializer_list<shared_model::interface::permissions::Role>);
```

Comment:
To prevent 503 Swig warnings you can add lines like
```
%rename(opx) shared_model::interface::PermissionSet<shared_model::interface::permissions::Role>::operator[];
%rename(opy) shared_model::interface::PermissionSet<shared_model::interface::permissions::Grantable>::operator[];
```
just above.

Reply:
That will create unnecessary names that can be used in bindings. Not really a desired behavior